en/flight_controller/intel_aero.md

```diff
@@ -28,9 +28,69 @@ OS_VERSION = Poky Aero (Intel Aero Linux Distro) 1.5.1-dev (pyro)"
 AIRMAP_VERSION = 1.8
 FPGA_VERSION = 0xc1
 ```
+## Setup Intel Aero using Ubuntu
 
+This chapter is based on the [Intel Aero wiki](https://github.com/intel-aero/meta-intel-aero/wiki/90-%28References%29-OS-user-Installation). Follow the steps to install the OS and Intel Aero repository (The Yocto does not necessarily have to be upgraded first). The required material for those steps consists of:
```

Comment:
Suggest modify this whole introduction slightly from top down to line 39 (just below the list). Specifically It would be good to link the specific sections in the wiki doc so it is *crystal* clear exactly what is run.  If you do it as below you don't need to mention that instructions are based on the Intel wiki, because that is obvious. 

So something like:

----

In order to install the OS and Aero you will need the following equipment: 	 
1. Power supply (battery or network cable)
1. Micro HDMI to HDMI cable to attach a monitor
1. Micro USB3 to USB2 female adapter
1. USB Hub to attach mouse and keyboard

Follow the linked instructions from [Intel Aero wiki > Installing Ubuntu on Intel Aero](https://github.com/intel-aero/meta-intel-aero/wiki/90-%28References%29-OS-user-Installation#installing-ubuntu-on-intel-aero):
1. [Upgrade Yocto first](https://github.com/intel-aero/meta-intel-aero/wiki/90-%28References%29-OS-user-Installation#upgrade-yocto-first) (Optional)
1. [OS](https://github.com/intel-aero/meta-intel-aero/wiki/90-%28References%29-OS-user-Installation#os)
1. [Intel Aero repository](https://github.com/intel-aero/meta-intel-aero/wiki/90-%28References%29-OS-user-Installation#os)

----

Source look like
```
In order to install the OS and Aero you will need the following equipment: 	 
1. Power supply (battery or network cable)
1. Micro HDMI to HDMI cable to attach a monitor
1. Micro USB3 to USB2 female adapter
1. USB Hub to attach mouse and keyboard

Follow the linked instructions from [Intel Aero wiki > Installing Ubuntu on Intel Aero](https://github.com/intel-aero/meta-intel-aero/wiki/90-%28References%29-OS-user-Installation#installing-ubuntu-on-intel-aero):
1. [Upgrade Yocto first](https://github.com/intel-aero/meta-intel-aero/wiki/90-%28References%29-OS-user-Installation#upgrade-yocto-first) (Optional)
1. [OS](https://github.com/intel-aero/meta-intel-aero/wiki/90-%28References%29-OS-user-Installation#os)
1. [Intel Aero repository](https://github.com/intel-aero/meta-intel-aero/wiki/90-%28References%29-OS-user-Installation#os)
```